rest/handler.go

```diff
@@ -150,7 +150,7 @@ func makeHandler(server *ServerContext, privs handlerPrivs, accessPermissions []
 // makeSilentHandler creates an http.Handler that will run a handler with the given method only logging at debug
 func makeSilentHandler(server *ServerContext, privs handlerPrivs, accessPermissions []Permission, responsePermissions []Permission, method handlerMethod) http.Handler {
 	return makeHandlerWithOptions(server, privs, accessPermissions, responsePermissions, method, handlerOptions{
-		httpLogLevel: base.Ptr(base.LevelDebug),
+		httpLogLevel: base.Ptr(base.LevelTrace),
```

Comment:
There's a `isSilentRequest()` method which needs changing too (this omits the event from audit log.

It feels like the log level should be a const referenced from both places - too easy to miss with them both referenced independently.